### DISCUSSION TITLE
SofaPython3 - Installation on Windows 11 - Module not found error

### DISCUSSION THREAD

**Discussion by TGeissenb** [sofa-framework/sofa#4566] [SofaPython3] [answered]
Hello,

I'm struggling to install the SofaPython3 plugin. If I open the GUI and navigate to "Edit->Plugin Manager->Add->"...\SOFA\v23.12.00\plugins\SofaPython3\bin\SofaPython3.dll", I get the following error:

![image](https://github.com/sofa-framework/sofa/assets/152016082/ff8b36d0-a6e3-4ac0-8d6b-a119b8e7f0a0)

I also tried to launch SOFA from Pycharm, but I get a similar error:
Traceback (most recent call last):
  File "C:\Users\tobias\PycharmProjects\SoFa_Simulation\test.py", line 2, in <module>
    import Sofa
  File "C:\Users\tobias\SOFA\v23.12.00\plugins\SofaPython3\lib\python3\site-packages\Sofa\__init__.py", line 140, in <module>
    import Sofa.Helper
ModuleNotFoundError: No module named 'Sofa.Helper'

I downloaded the SOFA v23.12.00 exe installer for Windows and followed the installation instructions (https://sofapython3.readthedocs.io/en/latest/content/Installation.html#setup-your-environment):
SOFA_ROOT=C:\Users\tobias\SOFA\v23.12.00
SOFAPYTHON3_ROOT=C:\Users\tobias\SOFA\v23.12.00\plugins\SofaPython3
PYTHON_ROOT=C:\Users\tobias\AppData\Local\Programs\Python\Python38
PYTHONPATH=C:\Users\tobias\SOFA\v23.12.00\plugins\SofaPython3\lib\python3\site-packages
Path=...;C:\Users\tobias\AppData\Local\Programs\Python\Python38;C:\Users\tobias\AppData\Local\Programs\Python\Python38\DLLs;C:\Users\tobias\AppData\Local\Programs\Python\Python38\Lib;C:\Users\tobias\SOFA\v23.12.00\bin

In the meantime, I tried using Python 3.12, 3.7 and now installed Python 3.8.10 as suggested in the installation manual.
Can you please help me resolve this issue with SofaPython3?

Thanks!
Best, Tobias

**Comment by bakpaul** [comment, marked as answer]
Hello, this is the 23.12 release and I see that you are using python3.8. Actualy, those binaries where compiled using python3.10. You should install it and modify your `Path`to point to this version.

**Reply by TGeissenb** [reply]
Thanks, that resolved the problem!
